ivy/functional/frontends/numpy/sorting_searching_counting/sort_complex.py

```diff
@@ -0,0 +1,55 @@
+import ivy
+from ivy.functional.frontends.numpy.func_wrapper import (
+    to_ivy_arrays_and_back,
+    handle_numpy_casting,
+)
+
+
+@handle_numpy_casting
+@to_ivy_arrays_and_back
+def sorting_complex_numbers(array, dtype=complex):
+    if dtype:
+        array = [ivy.astype(ivy.array(a), ivy.as_ivy_dtype(dtype)) for a in array]
+    return ivy.sort_complex(array, dtype=complex)

```

Comment:
ivy doesn't have a sort complex function and we don't yet support complex dtypes